README.md

```diff
@@ -1,82 +1,12 @@
-401 JS --  Lab 11 rest api using express
+401 JS --  Lab 08 rest api using http module
 ===
 
-## Submission Instructions
-  * Work in a fork of this repository
-  * Work in a branch on your fork
-  * Write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-duncan`
-  * Submit a pull request to this repository
-  * Submit a link to your pull request on canvas
-  * Submit a question, observation, and how long you spent on canvas  
-  
-## Learning Objectives  
-* Students will understand the purpose of a server side router 
-* Students will be able to implement a rest API using using nodes http module
-
 ## Resources  
-* [express docs](http://expressjs.com/en/4x/api.html)
-* [bodyparser docs](https://github.com/expressjs/body-parser)
-* [superagent docs](https://visionmedia.github.io/superagent/)
-* [http errors docs](https://github.com/jshttp/http-errors)
+* [Nodejs http docs]
```

Comment:
Good start for a README, but make sure to include more detailed instruction about your specific routes, i.e. ``A delete request to /api/thundercats/:id deletes the item with that specified id.``